code/modules/reagents/chemistry/machinery/chem_dispenser.dm

```diff
@@ -297,7 +297,7 @@
 
 /obj/machinery/chem_dispenser/wrench_act(mob/user, obj/item/I)
 	. = TRUE
-	if(!I.use_tool(src, user, 0, volume = I.tool_volume))
+	if(!I.use_tool(src, user, 4 SECONDS, volume = I.tool_volume))
```

Comment:
```suggestion
	if(!I.use_tool(src, user, time = 4 SECONDS, volume = I.tool_volume))
```

Reply:
use_tool uses delay not time, it even gave me an error while making this PR because i didnt notice that chem_dispenser was doing it differently.

Reply:
that would make sense, ignore my suggestion then